app/src/main/Member.js

```diff
@@ -438,6 +465,19 @@ export default class Member extends CoreMember {
         });
     }
 
+    /**
+     * Prepares a bulk transfer token, retrieves policy, and resolves payload.
+     *
+     * @param tokenPayload - token payload to prepare
+     * @returns a policy and the resolved payload
+     */
+    prepareBulkTransferToken(tokenPayload: Object): Promise<Object> {
```

Comment:
This is redundant since we have prepareToken right?

Reply:
wanted to be consistent with the java-sdk